x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/boosts/boosts.scss

```
@@ -0,0 +1,32 @@
+.boosts {
+  .boosts__header.euiPageContentHeader {
+    margin-bottom: 0;
+  }
+
+  .boosts__select {
+    min-width: $euiSizeXXL * 4;
+  }
+
+  .boosts__itemContent {
+    width: 100%;
+  }
+
+  .boosts__item {
+    margin-top: $euiSize;
+  }
+
+  .boosts__item + .boosts__item {
+    margin-top: $euiSizeS;
+  }
```

I think in this case we should opt to use `&`, since .boosts__item is already searchable

```suggestion
  .boosts__item {
    margin-top: $euiSize;

    & + & {
      margin-top: $euiSizeS;
    }
  }

```

---

boosts__item

---

https://github.com/elastic/kibana/pull/90621/commits/e7f6bf47eee03054c4d73e7ccb27e958e65685b6